chia/full_node/hint_store.py

```diff
@@ -38,6 +39,23 @@ async def get_coin_ids(self, hint: bytes, *, max_items: int = 50000) -> List[byt
             await cursor.close()
         return [bytes32(row[0]) for row in rows]
 
+    async def get_coin_ids_multi(self, hints: Set[bytes], *, max_items: int = 50000) -> List[bytes32]:
```

Comment:
I assumed this would be a big optimization from doing a bunch of individual queries, but I don't actually think it's that much faster interestingly. I'm pretty disappointed by our SQLite DB performance overall with the wallet protocol.

Reply:
ideally, we would have a unit test for this function as well

Reply:
Just added it